packages/toc/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "@jupyterlab/toc",
+  "version": "4.0.0",
+  "private": false,
+  "description": "Table of Contents extension for JupyterLab",
+  "keywords": [
+    "jupyterlab"
+  ],
+  "homepage": "https://github.com/jupyterlab/jupyterlab-toc",
```

Comment:
same comments as the other `package.json` update urls and description, and remove the private line